src/components/pages/StudentHome/StudentStoryCard.js

```diff
@@ -1,15 +1,65 @@
 import React, { useState } from 'react';
 import '../../../styles/StudentStyles/index.less';
 import { connect } from 'react-redux';
+import { Card } from 'antd';
+import { SendOutlined } from '@ant-design/icons';
 
 function StudentStoryCard(prop) {
   return (
-    <div className="student-story-card">
-      <img src="https://via.placeholder.com/50" alt="placeholder"></img>
-      <h3>Teacher Name</h3>
-      <h4>Coder Heroes</h4>
-      <h4>9 Minutes Ago</h4>
-    </div>
+    <Card className="student-story-card">
+      <div className="top-story-card">
```

Comment:
The top-story-card uses a div instead of Card component like the container, was this intentional or should it be changed to use AntDesign?

Reply:
I second this :) Card Components are the bomb!

Reply:
@brandon-bee This was intentional